src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/RequestForJudgementNotificationService.java

```diff
@@ -64,7 +64,7 @@ private EmailData prepareEmailData(Claim claim) {
         EmailAttachment ccjPdfAttachment = generateCountyCourtJudgmentPdf(claim);
 
         return new EmailData(emailProperties.getCountyCourtJudgementRecipient(),
-            "J default judgement request " + claim.getReferenceNumber(),
+            "J judgement request " + claim.getReferenceNumber(),
```

Comment:
This should be "J judgment request [case num]" as per A/C.
Please also correct the spelling of "Judgment" (judgement is the US spelling) in any new code / files. (Also renaming existing files if you fancy.)